simplq/src/store/queueInfo/queueInfoSlice.js

```diff
@@ -13,6 +13,13 @@ const queueInfoSlice = createSlice({
     [getQueueInfoByName.fulfilled]: (state, action) => {
       return action.payload;
     },
+    [deleteQueue.fulfilled]: (state, action) => {
+      const { queueId } = action.payload;
+      if (state.queueId === queueId) {
+        return {};
```

Comment:
let us just set the status to deleted inside the if condition

Reply:
Done